### PR TITLE
Set default page size to A4

### LINKED ISSUE #319: Default pdf-page-size
Hello,

the current default `pdf-page-size` is `US Letter`. Most of the world uses the `A` format (most commonly `A4`). I would like to discuss, if we could introduce this as the default format in `asciidoctor-pdf`.

Of course, one format has to be the default, and everyone prefers the one that she uses on a daily bases. I think going for the ISO standard is a reasonable choice.

Also, the property that the `A` format scales nicely to bigger and smaller sheets (two `A5` sheets being exactly one `A4`, etc, due to the aspect ratio) makes it superior format (in my completely biased point of view, at least).

Any thoughts on that?

### PR DESCRIPTION
**Pull request by stephenhay**
Resolves #319.

Visually confirmed in Evince: File > Properties.

### CODE REVIEW

File: lib/asciidoctor-pdf/converter.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# encoding: UTF-8
+# encoding: utf-8
```
Comment (veeg) [inline lib/asciidoctor-pdf/converter.rb]: This probably shouldn't be part of this PR.

Comment (mojavelinux) [inline lib/asciidoctor-pdf/converter.rb]: I agree. Ruby stores encoding names internally in uppercase, so technically UTF-8 is more correct (though both cases work).

**Comment by mojavelinux** [comment]
Thanks! The change to the page_size is correct. Could you revert the encoding line, amend the commit (`git commit --amend lib/asciidoctor-pdf/converter.rb`), then force push the branch (`git push --force`)?

File: lib/asciidoctor-pdf/converter.rb (hunk 1 above)
Comment (stephenhay) [inline lib/asciidoctor-pdf/converter.rb]: Apologies. My editor changed that automatically and I missed it.

**Comment by stephenhay** [comment]
This has been done.

**Comment by mojavelinux** [comment]
Nice work again @stephenhay!
